Rmd/DESeq2_report.rnaseq.Rmd

```diff
@@ -1331,6 +1334,10 @@ write.table(Counts,
             file = file.path(paths$DEG_output,
                              paste0(NORM_TYPE, "-Per_sample_normalized_counts.txt")),
             quote = F, sep = '\t', col.names = NA)
+write.table(degSummary,
+            file = file.path(paths$DEG_output,
+                             paste0(NORM_TYPE, "-DEG_summary.txt")),
+            quote = F, sep = '\t', col.names = FALSE, row.names = FALSE)
```

Comment:
I can't really tell from these snippets but does this print column headers at all? Oh wait, it's just contrast names and # DEGs, never mind.